external/buildscripts/build_all_osx.pl

```diff
@@ -110,7 +110,7 @@
 
 	for my $file ('mono')
 	{
-		MergeIntoFatBinary("$distDirSourceBinX64/$file", "$distDirSourceBinARM64/$file", "$distDirDestinationBin/$file");
+		MergeIntoFatBinary("$distDirSourceBinX64/$file", "$distDirSourceBinARM64/$file", "$distDirDestinationBin/$file", 1);
```

Comment:
Long term should we keep fat binaries? Or split our binaries per arch?

Reply:
Honestly, I don't know. Embed runtimes are already split, but splitting mono executable (and libMonoPosixHelper.dylib for the executable) is much more tricky. There are a lot of places that assume hardcoded locations for these binaries. It took us years to get those fixed up when we were bringing up the Linux editor. Merging them here was just an easy way out.